### ISSUE TITLE
Bleemsync doesn't work with more than 3 Games.

### ISSUE THREAD

**Issue by Nostalgicnerd** [pathartl/BleemSync#126]
Hello, I ran into following issue.
I tried 5 different USB Sticks, with varying capacitys and types.
Im facing this issue unrelated if I use a 3.0 or 2.0 Stick.
What happens when using for example 4 or more than 4 games: 
1.Sony Logo
2.Healthy information screen
3.Blackscreen following a no signal and the Playstation classic shutting down into Standby.
This is not the case when using only 3 or less Games.
To make sure, I did various test, with different games.
I also tried different bleemsync versions with the newest release I also tried deleting the system/bleemsync folder
I also factory reset my Psclassic not changing anything.
The Video below showcases the problem.
https://youtu.be/e_ydcO4bNBQ
Btw its a PS1 bought in Germany,
Logs attached.
[ui_menu.log](https://github.com/pathartl/BleemSync/files/2706449/ui_menu.log)
[lolhack.log](https://github.com/pathartl/BleemSync/files/2706450/lolhack.log)

**Comment by alucard73** [comment]
Hello
like my neighbor I have the same problem. I have a MicroSD 128GB, until now it worked, but as soon as I put more than 70 games I have a black screen and the console goes out

**Comment by raynebc** [comment]
I thought I was making mistakes when I couldn't get my PS classic to launch from USB with 20 games and the console would display the Sony logo and power off.  I'll subscribe to this ticket and wait for a fixed release.

To the developer:  If I can help by performing any particular testing or providing some kind of logging (if BleemSync performs any), I'd be happy to do so.  If I manage to find that a specific game image is associated with the console crashing and its removal allows BleemSync to work as expected, I'll post an update.

**Comment by iRyanKGT** [comment]
You guys should test your games one by one to see which is the problematic file.
Also verify that the folder/file structure that BleemSync but build matches up to the readme.

Myself and countless other have 30+ games working without issue.

**Comment by FyreAnt** [comment]
Put the stick back into the computer, go to "my computer" right click and go to the properties of the stick, go to Tools tab, and under Error Checking, click "Check". I had the same problem. The issue is happening because when you turn the system off, its not properly closing the connection to the USB port and is simulating you pulling the stick out of the computer without properly ejecting it. Thus, some files are getting corrupted. Unfortunately, I haven't found a way to alleviate this issue without having to rebuild the stick after about playing 20 games or so. :(

**Comment by raynebc** [comment]
iRyanKGT:  Nostalgicnerd indicated testing different games and indicated that it's only malfunctioning when using more than 3 games regardless of what they are.  It's possible that game images themselves aren't the culprits, but I will do some testing when I get time.

FyreAnt:  The PS Classic console itself is crashing (powering off), and that is causing an unclean dismount of the file system.  This causes file system errors regardless of how we users are manually powering off the console or unplugging the flash drive.

**Comment by raynebc** [comment]
After testing, I narrowed it down to what seems to be one game (Grandia) that causes the console to crash (during the Playstation logo screen, the power status light alternates between green and amber, then the console turns off, then the power status light returns to amber).  It doesn't seem like the logs folder on the flash drive typically has contents after the crash has occurred.  Is there a good way to troubleshoot why this is happening?  The cue sheet looks correct, has the right name for the bin file.  Is the binary image for the game corrupt in some way, or could it be incompatible with the console?

**Comment by Nostalgicnerd** [comment]
I also noticed that my drives are getting corrupted after a bit of use, so I used chkdsk /f to fix them ofcourse formatting it again, and recopy the games. Yeah the crash when using more than 4 games for example corrupts the sticks aswell. My current setup uses 6 different usb devices to store the games.
I'd wish this issue was user error, but It's to my current pov not.
If I can provide any help to solve this I'm willing to do so.
